docs/_translations.md

```diff
@@ -11,6 +11,7 @@
 - [Chinese](/i18n/chinese/index.md)
 - [English](/index.md)
 - [Español](/i18n/espanol/index.md)
+- [Japanese](/i18n/japanese/index.md)
```

Comment:
We can enable this now, but before the download, this points to 404.

Reply:
Once this is merged I plan to manually run the download.